<launch>
<group ns="iris_0">
  <arg name="mav_name" default="iris"/>
  <arg name="command_input" default="1" />
  <arg name="gazebo_simulation" default="true" />

  <node pkg="geometric_controller" type="geometric_controller_node" name="geometric_controller" output="screen">
  		    <param name="mav_name" type="string" value="$(arg mav_name)" />
          <remap from="command/bodyrate_command" to="mavros/setpoint_raw/attitude"/>
          <remap from="planning/target_exist" to="/drone_0_ego_planner_node/planning/target_exist"/>
          <param name="ctrl_mode" value="$(arg command_input)" />
          <param name="enable_sim" value="$(arg gazebo_simulation)" />
	        <param name="velocity_yaw" value="false" />
      	  <param name="yaw_heading" value="0.0" />
          <param name="max_acc" value="10.0" />
          <!-- <param name="Kp_x" value="1.5" />
          <param name="Kp_y" value="1.5" />
          <param name="Kp_z" value="1.5" />
          <param name="Kv_x" value="1.5" />
          <param name="Kv_y" value="1.5" />
          <param name="Kv_z" value="1.5" /> -->
          <param name="Kp_x" value="3.0" />
          <param name="Kp_y" value="3.0" />
          <param name="Kp_z" value="7.0" />
          <param name="Kv_x" value="3.0"/>
          <param name="Kv_y" value="3.0"/>
          <param name="Kv_z" value="3.3"/>
          <param name="normalizedthrust_offset" value="0.0" />
          <param name="normalizedthrust_constant" value="0.0755102" />
          <param name="attctrl_constant" value="0.3" />
          <param name="init_pos_x" value="0.0" />
          <param name="init_pos_y" value="0.0" />
          <param name="init_pos_z" value="1.0" />
  </node>
</group>
  <!-- Launch rqt_reconfigure -->
  <!-- <node pkg="rqt_reconfigure" type="rqt_reconfigure" output="screen" name="rqt_reconfigure" /> -->
</launch>
